### PR TITLE
Temperature from gpx atemp extension

### PR DESCRIPTION
**Pull request by xneek**
Based on https://github.com/Raruto/leaflet-elevation/issues/205

I fixed it a little and tested it on different sets of real data - it works

image

### CODE REVIEW

**Comment by Raruto** [comment]
Hi @xneek,

check out my latest edits and let me know

👋 Raruto

File: src/handlers/temperature.js [hunk 1]
```diff
@@ -6,21 +6,19 @@ export function Temperature() {
...
 		decimals: 2,
-		pointToAttr: (point, i) => {
-			let tempVal = point.meta.atemps ?? point.meta.atemps ?? point.prev('temperature');
-			return L._(
-				opts.imperial
-					? (tempVal * 9 / 5) + 32 // Fahrenheit = (Celsius * 9/5) + 32
-					: tempVal
-			);
-		},
+		pointToAttr: (point, i) => (point.meta.atemps ?? point.meta.atemps ?? point.prev('temperature')) * opts.temperatureFactor1 + opts.temperatureFactor2,
```
Comment (xneek) [inline src/handlers/temperature.js:21]:
```suggestion
		pointToAttr: (point, i) => (point.meta.atemps ?? point.prev('temperature')) * opts.temperatureFactor1 + opts.temperatureFactor2,
```

**Comment by xneek** [comment]
Hi, @Raruto. I discovered a small pointless duplication of code - added a suggestion for correction. Everything else is good

File: src/options.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export var Options = {
 	followMarker: true,
 	imperial: false,
 	legend: true,
-	handlers: ["Distance", "Time", "Altitude", "Slope", "Speed", "Acceleration"],
+	handlers: ["Distance", "Time", "Altitude", "Slope", "Speed", "Acceleration", 'Temperature'],
```
Comment (Raruto) [inline src/options.js:21]: @xneek I need to double check the example, but I don't think I'll keep it as default (eg. like `pace` or `linear-gradient` handlers)

**Comment by Raruto** [comment]
Released in **v2.5.1**

**Pull request merged**